libbpf-rs/src/btf.rs

```diff
@@ -0,0 +1,440 @@
+//! Parse and introspect btf information, from files or loaded objects.
```

Comment:
Let's create a proper module structure below `src/btf/`? I find it confusing to have a sub-directory without `mod.rs` and don't believe it's encouraged Rust project layout. E.g.,
`src/btf.rs` -> `src/btf/mod.rs`

Reply:
this layout was actually introduced as a replacement for the `mod.rs` layout. I personally don't have much strong feelings about either one, iirc the reason for this layout is that it's better for people using fuzzy finders in their IDE (typing mod.rs wasn't usefull because it showed every mod) but I use a fuzzy finder and I can work the both layouts so it never really bothered me. If you prefer `src/btf/mod.rs` I can change it

Reply:
mine does too, best fuzzy is just `find | fzf` or equivalent xD I'm gonna go the mod.rs route. I've personally tried giving the non `mod.rs` route but I think it looks more confusing when you're looking at the dir structure